### PR TITLE
Add AT Reference variables

### PR DESCRIPTION
**Pull request by LeilaZarban**
- [ ] You've read the [Contributor Guide](https://github.com/fiskaltrust/.github/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/fiskaltrust/.github/blob/main/CODE_OF_CONDUCT.md).
- [ ] You've included unit or integration tests for your change, where applicable.
- [ ] You've included inline docs for your change, where applicable.
- [ ] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

{Summary of the changes}

## Description

{Detail}

Fixes #{issue number}

### CODE REVIEW

File: _markets/at/technical-operations/rollout-automation/templates/_variables.mdx [hunk 1]
```diff
@@ -1 +1,64 @@
...
+
+The actual attribute you'd like to reference. Currently, `id` and `url` are supported.
+
+
+
+#### Example
+
+:::info Default field with shop-bought SCUs
+
+Any TSE bought via fiskaltrust.Shop or 'Rollout Management' can have an incrementing integer field `counter` assigned.
```
Comment (wrueting) [inline _markets/at/technical-operations/rollout-automation/templates/_variables.mdx:42]: SCU, the term TSE only exists in germany, doesn't exist in AT

File: _markets/at/technical-operations/rollout-automation/templates/_variables.mdx [hunk 2]
```diff
@@ -1 +1,64 @@
...
+
+:::
+
+Let's say you have purchased two onlinescu TSEs for one outlet via fiskaltrust.Shop. In that case, their SCUs can have a `counter` field with the value of `1` and '2'. Using a reference variable in your template, you can now reference these SCUs (in particular their IDs, in this example) by using the following variable.
```
Comment (wrueting) [inline _markets/at/technical-operations/rollout-automation/templates/_variables.mdx:46]: same as above TSE --> SCU

File: _markets/at/technical-operations/rollout-automation/templates/_variables.mdx (hunk 1 above)
Comment (LeilaZarban) [reply _markets/at/technical-operations/rollout-automation/templates/_variables.mdx]: done

File: _markets/at/technical-operations/rollout-automation/templates/_variables.mdx (hunk 2 above)
Comment (LeilaZarban) [reply _markets/at/technical-operations/rollout-automation/templates/_variables.mdx]: done

**Review by wrueting — APPROVED** [review]

**Pull request merged**